index.html

```diff
@@ -0,0 +1,151 @@
+        <h2>Is There Such A Thing As Too Much Social Proof?</h2>
+      </article>
+      <article class="thumbnail light-grey">
+        <h2>15 Questions To Ask Your Next Potential Employer</h2>
+      </article>
+      <article class="thumbnail light-grey">
+        <h2>How AI Is Helping Solve Climate Change</h2>
+      </article>
+      <div class="aside">
+        <div class="thumbnail-aside darkgrey-black">
+          <h2>
+            My Design Process Of The Cover Design For Smashing Magazine Print
+            Issue #1
+          </h2>
+        </div>
+      </div>
+      <div class="big white-black">
+        <img
+          src="https://d33wubrfki0l68.cloudfront.net/72280e1450d4ecfc45ef96ea00b27b4bf4ef5d06/eb759/images/smashing-cat/cat-smashing-tv-2.svg"
+          alt="image"
+        />
+        <h2>New on Smashing TV</h2>
+      </div>
+    </div>
+    <div class="full-width type2 light-grey ">
+      <h2>Smashing Print #1: Ethics & Privacy</h2>
+      <img
+        src="https://d33wubrfki0l68.cloudfront.net/148069bc1849f0c7b2304c945073dbdfdb6341f0/b12bb/images/printed-mag/printmag-homepage-opt.svg"
+        alt="image"
+      />
+    </div>
+    <div class="type3">
+      <div class="wide typ3-content darkgrey-black">
+        <h2>Person Of The Week</h2>
+      </div>
+      <article class="thumbnail grey-black">
+        <h2>Investigating Animation Performance</h2>
+      </article>
+      <article class="thumbnail grey-black"><h2>Design Tips</h2></article>
+      <article class="thumbnail light-grey">
+        <h2>Chasing Waterfalls</h2>
+      </article>
+      <article class="thumbnail grey-black"><h2>AST Explorer App</h2></article>
+      <article class="thumbnail light-grey">
+        <h2>How To Write Better JavaScript</h2>
+      </article>
+      <article class="thumbnail grey-black">
+        <h2>Privacy Issues Newsletter</h2>
+      </article>
+    </div>
+    <div class=" full-width type4 light-grey ">
+      <h2>Design Systems</h2>
+    </div>
+    <div class="type1-container">
+      <article class="thumbnail light-grey">
+        <h2>New Front-End Challenges To Master</h2>
+      </article>
+      <article class="thumbnail light-grey">
+        <h2>Performance Optimization & Strategy</h2>
+      </article>
+      <article class="thumbnail light-grey">
+        <h2>Improving Workflow & Data Protection</h2>
+      </article>
+      <div class="aside">
+        <div class="thumbnail-aside darkgrey-black">
+          <h2>Design Patterns & Checklists</h2>
+        </div>
+      </div>
+      <div class="big white-black">
+        <h2>Smashing Newsletter</h2>
+        <img
+          src="https://d33wubrfki0l68.cloudfront.net/ce7741e1659c0366fda51555365194e0bfd136c0/ae578/images/smashing-cat/cat-with-trampet.svg"
+          alt="image"
+        />
+      </div>
+    </div>
+    <footer class=" full-width footer-container grey-black">
+      <h2>Browse All Smashing Magazine Topics</h2>
+    </footer>
```

Comment:
Your footer should have **centered links**. Please make them visible by adding some dummy text like ```link-1``` or anything you like. This **[centered links]** is a requirement for the completion of this project.